analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/NonStandardCryptographicAlgorithmsShouldNotBeUsedTest.cs

```diff
@@ -33,28 +33,28 @@ public class NonStandardCryptographicAlgorithmsShouldNotBeUsedTest
         [TestMethod]
         [TestCategory("Rule")]
         public void NonStandardCryptographicAlgorithmsShouldNotBeUsed_CSharp() =>
-            Verifier.VerifyAnalyzer(@"TestCases\NonStandardCryptographicAlgorithmsShouldNotBeUsed.cs",
+            Verifier.VerifyAnalyzer(@"TestCases\Hotspots\NonStandardCryptographicAlgorithmsShouldNotBeUsed.cs",
```

Comment:
While looking at the files in the directory, this rule has too long name. Let's rename this type and all related stuff to `UsingNonstandardCryptography`